src/Documents.tsx

```diff
@@ -40,6 +44,28 @@ export default function Documents(props: DocumentsProps) {
     [routerPanesState, groupIndex, handleEditReference]
   )
 
+  const handleUnsetClick = useCallback(
+    (id: SanityDocument['_id'], referenceId: SanityDocument['_id']) => {
+      const documentId = id.replace('drafts.', '')
+
+      client
+        // TODO: does this need to account for a draft?
```

Comment:
I'd rather see you figure this todo out before mreging, it will just leave use wondering in the future.